src/extension/ln/nostr/types.ts

```diff
@@ -16,3 +16,10 @@ export enum EventKind {
   DM = 4,
   Deleted = 5,
 }
+
+export type Nip26DelegateConditions = {
```

Comment:
It looks like multiple kinds can be passed at once. Is this something we want to support?

Reply:
yeah, although last I heard the PR for multiple kinds looks like it might get rolled back, definitely we should support it